src/OpenTelemetry/Metrics/Exemplar/Exemplar.cs

```diff
@@ -18,10 +18,11 @@
 
 namespace OpenTelemetry.Metrics
 {
+#pragma warning disable SA1623 // The property's documentation summary text should begin with: `Gets or sets`
     /// <summary>
     /// Represents an Exemplar data.
     /// </summary>
```

Comment:
Weird.. the style cops didn't catch this before? 👮‍♂️

Reply:
It complains about it only when I mark the struct internal.